examples/txDeposit.ts

```diff
@@ -15,6 +15,7 @@ const main = async () => {
 
     // # 2 [OPTIONAL] If you did not pass a mnemonic, you can retrieve it
     const mnemonic = user.getNightfallMnemonic();
+    console.log("MNEMONIC: ", mnemonic);
```

Comment:
I did not console log the `mnemonic` to reinforce the idea of keeping it private, can you please remove? :upside_down_face:

Reply:
Removed.